SVA/lib/secmem.c

```diff
@@ -502,7 +503,7 @@ freeSecureMemory (void) {
 }
 
 void
-sva_ghost_fault (uintptr_t vaddr) {
+sva_ghost_fault (uintptr_t vaddr, unsigned long code) {
```

Comment:
Could you add a function-level comment for sva_ghost_fault()?  I neglected to write one when I added sva_ghost_fault().  :(